app/server/src/features/organizations/resolvers.ts

```diff
@@ -10,6 +12,38 @@ const toUserId = toGlobalId('User');
 const toEventId = toGlobalId('Event');
 
 export const resolvers: Resolvers = {
+    Query: {
+        async isOrganizer(parent, args, ctx, info) {
+            if (!ctx.viewer.id) {
+                return false;
+            }
+            if (process.env.NODE_ENV === 'development') return true;
+            const email = await Organization.findEmailByViewerId(ctx.viewer.id, ctx.prisma);
+            const bucketName = 'organizers';
+            const fileName = 'list-of-organizers.csv';
+            const storage = new Storage({
+                projectId: process.env.GCP_PROJECT_ID,
+                keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
+            });
+            const options = {
+                destination: './downloadedFile.txt',
+            };
+            const file = storage.bucket(bucketName).file(fileName);
+            let found = false;
+            try {
+                await file.download(options);
+            } catch (err) {
+                throw new Error(err);
+            }
+            const allFileContents = fs.readFileSync('./downloadedFile.txt', 'utf-8');
+            allFileContents.split(/\r?\n?,/).forEach((line) => {
+                if (line === email?.email) {
+                    found = true;
+                }
+            });
+            return found;
```

Comment:
I think a good chunk of this logic could be delegated to another Organizer method.  Lets say `Organizer.isOnOrganizerList`

Reply:
done!